deploy/eck-fleet-server/Chart.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: eck-fleet-server
+description: A Helm chart to deploy Elastic Fleet Server as an Agent managed by the ECK Operator.
```

To keep it simple and consistent with its [doc](https://www.elastic.co/guide/en/fleet/current/fleet-overview.html), I think we should just write 'Fleet Server' and not 'Fleet Server Agent' or 'Elastic Fleet Server Agent'. It is not as ambiguous as Agent.

```suggestion
description: A Helm chart to deploy Fleet Server managed by the ECK Operator.
```

(`grep '\(Elastic \)*Fleet Server Agent' -ir deploy` to list occurrences to change)

---

Sounds good.  All have been updated.